azkaban-common/src/main/java/azkaban/executor/AbstractExecutorManagerAdapter.java

```diff
@@ -127,6 +127,16 @@ public ExecutableFlow getExecutableFlow(final int execId)
     return this.executorLoader.fetchExecutableFlow(execId);
   }
 
+  @Override
+  public void preloadExecutionOptions(ExecutableFlow exFlow) throws ExecutorManagerException {
+    try {
+      exFlow.setFlowParamsFromProps(this.projectManager.loadPropsForExecutableFlow(exFlow));
```

Comment:
4. and in `AbstractExecutorManagerAdapter`, use `projectManager` to load the Props